src/main/java/JavaTwo/LabTwo/MainTest.java

```diff
@@ -8,23 +8,27 @@
 
 public class MainTest {
 
-
-    public static void main(String[] args) throws IOException {
-        String excelFilePath = "utils/sudoku.xlsx";
-        FileInputStream inputStream = new FileInputStream(new File(excelFilePath));
-        Workbook workbook = new XSSFWorkbook(inputStream);
-        workbook.close();
-        SudokuBoardChecker sudokuBoardChecker = new SudokuBoardChecker(workbook);
-
-
-        for (int i =0; i<7; i++){
-            System.out.println("Board" + i);
-            System.out.println(sudokuBoardChecker.verifyBoard(i));
-        }
-
-
-
-
-    }
+	public static void main(String[] args) {
+		String excelFilePath = "utils/sudoku.xlsx";
+		try {
+			FileInputStream inputStream = new FileInputStream(new File(excelFilePath));
+			Workbook workbook = new XSSFWorkbook(inputStream);
+			workbook.close();
+			SudokuBoardChecker sudokuBoardChecker = new SudokuBoardChecker(workbook);
+			for (int i = 0; i < 7; i++) {
+				System.out.println("Board" + i);
+				System.out.println(sudokuBoardChecker.verifyBoard(i));
+			}
+
+		} catch (FileNotFoundException ex1) {
```

Comment:
First of all we should be worried about FileNotFoundException cause simply one IOException is too general.